samples/tab-add-in-combined/nodejs/appManifest/manifest.json

```diff
@@ -61,7 +61,6 @@
         "messageTeamMembers"
     ],
     "validDomains": [
-        "${{ADDIN_DOMAIN}}",
         "${{TAB_DOMAIN}}"
```

Comment:
This should be auto generated?

Reply:
If yes, please make sure to use proper place holder.

Reply:
${{TAB_DOMAIN}} is auto generated. ${{ADDIN_DOMAIN}} we are not using.

Reply:
Fixed